pom.xml

```diff
@@ -599,7 +599,11 @@
                 <artifactId>org.wso2.carbon.directory.server.manager.ui</artifactId>
                 <version>${project.version}</version>
             </dependency>
-
+            <dependency>
```

Comment:
Do we need to do this change in the root pom file ?

Reply:
Updated in